### PR TITLE
Convert all js code to coffee script

### PR DESCRIPTION
**Pull request by Manisha-Darade**
Hello team,

This PR converts  JavaScript files to CoffeeScript.
Ticket link : https://trello.com/c/wTLrkl89/31-2-convert-all-js-code-to-coffee-script

Please review it and let me know if any further changes are required.
Thanks

### CODE REVIEW

**Comment by rohandaxini** [comment]
@Manisha-Darade You'll need to rebase this branch with `master` branch. Could you please do so? Also can you both @Darshan-Chauhan and you make it a practices to rebase branches often - I have to repeat it several times for every PR.

cc: @sampatbadhe @rohanpujaris

**Comment by Manisha-Darade** [comment]
Sure @rohandaxini . I will do it.

File: app/assets/javascripts/common.js.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+@replaceTag = (data) ->
+  sanitizedData = data.replace(/<code>/g, '').replace(/<\/code>/g, '')
+  sanitizedData
```
Comment (rohanpujaris) [inline app/assets/javascripts/common.js.coffee]: Do we need `sanitizedData` at last line?

Comment (Manisha-Darade) [inline app/assets/javascripts/common.js.coffee]: No, we don't need it. function always return last value. will correct it @rohanpujaris

**Comment by rohanpujaris** [comment]
:+1:

**Pull request merged**